backend/app/controllers/practitioner/session-frames.js

```diff
@@ -1,18 +1,5 @@
 const prisma = require('../prisma-client');
 
-const create = async (ctx) => {
-  const sessionFrame = await prisma.sessionFrame.create({
-    data: {
-      ...ctx.request.body
-    }
-  });
-
-  ctx.body = {
-    data: sessionFrame
-  };
-  ctx.status = 200
-}
-
 const get = async (ctx) => {
   const id = parseInt(ctx.params.id);
```

Comment:
If we support this functionality (which I'm not even sure we need to be able to query for an individual sessionFrame), we should verify the session a sessionFrame belongs to is associated with the practitioner making the request

Reply:
Not sure if we will keep this long term but I figured I would leave it for now for debugging.